connect/config.py

```diff
@@ -44,7 +44,7 @@ def __init__(
             if not os.path.exists(file):
                 raise IOError('No file `{}` on directory'.format(file))
 
-            with open(file) as config_file:
+            with open(file, 'r') as config_file:
```

Comment:
what is the reason to add 'r' to all open()? (it is well-known default ...)

Reply:
Uhmm you're right, I don't know what I was thinking here. I'll reverse this commit.